### ISSUE TITLE
OSError: [Errno 8] Exec format error

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#103]
Zeus version:
`1.0.49.ced6`

Firefox version:
`((52, 4))`

Geckodriver version:
`geckodriver-v0.19.0-linux32.tar.gz`

Error info:
```Traceback (most recent call last):
  File "/home/Scanner/Zeus-Scanner/var/google_search/search.py", line 242, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/Scanner/Zeus-Scanner/var/google_search/search.py", line 129, in get_urls
    browser = webdriver.Firefox(profile, proxy=proxy_to_use)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/webdriver.py", line 144, in __init__
    self.service.start()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/common/service.py", line 74, in start
    stdout=self.log_file, stderr=self.log_file)
  File "/usr/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 8] Exec format error
````

Running details:
`Linux-3.18.31-perf-g0bf156d-00991-gb62734d-aarch64-with-Kali-kali-rolling-kali-rolling`

Commands used:
`zeus.py -d https://m.lenovo.com.cn`

Log file info:
```2017-10-16 13:43:33,759;zeus-log;INFO;log file being saved to '/home/Scanner/Zeus-Scanner/log/zeus-log-8.log'...
2017-10-16 13:43:33,760;zeus-log;INFO;using default search engine...
2017-10-16 13:43:33,760;zeus-log;INFO;starting dork scan with query 'https://m.lenovo.com.cn'...
2017-10-16 13:43:33,761;zeus-log;INFO;attempting to gather query URL...
2017-10-16 13:43:33,936;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-16 13:43:33,937;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-16 13:43:33,966;zeus-log;ERROR;[7;31;31msearch.pyc failed to gather the URL from search engine, caught exception '[Errno 8] Exec format error' exception has been logged to current log file...
Traceback (most recent call last):
  File "/home/Scanner/Zeus-Scanner/var/google_search/search.py", line 242, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/Scanner/Zeus-Scanner/var/google_search/search.py", line 129, in get_urls
    browser = webdriver.Firefox(profile, proxy=proxy_to_use)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/webdriver.py", line 144, in __init__
    self.service.start()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/common/service.py", line 74, in start
    stdout=self.log_file, stderr=self.log_file)
  File "/usr/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 8] Exec format error

2017-10-16 13:43:35,280;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-16 13:43:35,280;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-16 13:43:35,292;zeus-log;INFO;[32mextracting traceback from log file...[0m
2017-10-16 13:43:35,292;zeus-log;INFO;[32mattempting to get firefox browser version...[0m
```

**Comment by Ekultek** [comment]
That's weird, this is a wrong driver issue again.. Give me a few minutes.

**Comment by Ekultek** [comment]
Patched via fc782c7. I decided to use a different installation method, remove the geckodriver from `/usr/bin` and reinstall Zeus, you should be fine after that. You can always just upgrade firefox to the newest version as well.
